core/src/main/scala/kafka/log/FileMessageSet.scala

```diff
@@ -430,7 +430,7 @@ class FileMessageSet private[kafka](@volatile var file: File,
 
 }
 
-object FileMessageSet
+object FileMessageSet extends Logging
```

Comment:
This changes the logger name slightly (the class of the companion object instead of the actual class). We could override `loggerName` to maintain compatibility.